content/en/docs/components/hyperparameter-tuning/resume-experiment.md

```diff
@@ -5,8 +5,9 @@ weight = 40
                     
 +++
 
-This page describes in detail how to modify running experiment and restart succeeded experiment.
-Follow this guide to known more about changing experiment execution process and use various
+This page describes in detail how to modify running experiment
+and restart succeeded experiment. Follow this guide to know more
+about changing the experiment execution process and use various
```

Comment:
- "a running experiment" or "running experiment"
- "page" -> "guide" (everything can be a page?)
- "succeeded"? -> "completed" (success in ML experiments can be subjective, but completion (it's over) is more objective, I think)
- Talk to the reader - "you will learn more..."

```suggestion
This guide describes how to modify running experiments
and restart completed experiments. You will learn
about changing the experiment execution process and use various
```